download_starlink_TLE1.1.py

```diff
@@ -1,31 +1,35 @@
 #for downloading TLEs on a regular timescale
 
-import pandas as pd
 import requests
 from datetime import date
 import os.path
 
+# creates date
 today = date.today()
 # dd/mm/YY
-d1 = today.strftime("%d/%m/%Y")
-# print("d1 =", d1)
+d1 = today.strftime("%d-%m-%Y")
 
+
+# takes Url and downloads into bytes(?)
 url = 'https://www.celestrak.com/NORAD/elements/starlink.txt'
 req = requests.get(url)
 url_content = req.content
-txt_file = open('starlink_TLEdata.txt', 'wb')
-txt_file.write(url_content)
-# txt_file.close()
 
-save_path = 'C:\\Users\\rasta\\Documents\\TLEs'
-file_name = 'starlink_TLEdata.txt'
-completeName = os.path.join(save_path, file_name)
+#adds timstamp to name, and directs to the right file
+with open(f'starlink_TLEdata-{d1}.txt', 'wb') as txt_file:

```

Comment:
You should put your complete filename up here. Then you only have to open this one file.

```suggestion
save_path = 'C:\\Users\\rasta\\Documents\\TLEs'
save_file = f'starlink_TLEdata-{d1}.txt'
completeName = os.path.join(save_path, file_name)
with open(completeName, 'wb') as txt_file:

```